.github/workflows/reusable-prober.yml

```diff
@@ -191,9 +183,11 @@ jobs:
           docker pull ghcr.io/sigstore/sigstore-probers:latest
           # the last argument in the next command is not used, it is required because the container doesn't have a default command
           docker create --name binaries ghcr.io/sigstore/sigstore-probers /usr/local/bin/crane
-          docker cp binaries:/usr/local/bin/cosign /usr/local/bin/
           docker cp binaries:/usr/local/bin/crane /usr/local/bin/
 
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.10.0
```

Comment:
What's the version of the cosign binary in the container? Would be a little faster if we can reuse that rather than installing cosign separately.